docs/images/drowio/AKS_ScalarDB_Server_App_In_Cluster.drawio

```diff
@@ -0,0 +1,329 @@
+<mxfile host="app.diagrams.net" modified="2023-06-23T05:03:08.642Z" agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36" etag="T3Ayco6tUFRY6ghw4wqN" version="21.5.0" type="google">
```

Comment:
The `.drowio` file is XML file. By adding this XML file in this repository, we can manage the diagram (PNG file) as a code. 

If you want to update/fix the diagram:

1. Import this XML file to [Drowio](https://app.diagrams.net/).
2. Update the diagram on the GUI of Drowio.
3. Export the updated diagram as both PNG and XML files.
4. Override PNG and XML files in this repository.